### DISCUSSION TITLE
Adicionar menus bootstrap ao site

### DISCUSSION THREAD

**Discussion by VictorCapoeira** [community/community#146035] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Como adicionar menus no meu site utilizando bootstrap? 
Gostaria de adicionar uma barra de menu ao site que possibilite ser responsivo.

**Comment by marquescharlon** [comment, marked as answer]
Olá Victor, bom dia!

Você poderá localizar o componente navbar do Bootstrap em https://getbootstrap.com/docs/5.3/components/navbar/
Basta copiar o melhor que se adequa ao seu projeto.

Além disso, certifique-se das dependências do Bootstrap estarem configuradas em seu site.

- *Adicionar dentro da tag <head>:* 
`<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">`

- *Adicionar antes de encerrar a tag </body>:* 
`<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script> `
